roles/etcd/tasks/main.yml

```diff
@@ -2,19 +2,26 @@
 - name: Install etcd
   yum: pkg=etcd state=installed disable_gpg_check=yes
 
-- name: Install etcdctl
-  yum: pkg=etcdctl state=installed disable_gpg_check=yes
+#- name: Install etcdctl
+#  yum: pkg=etcdctl state=installed disable_gpg_check=yes
 
 - name: Write etcd global config file
   template: src=etcd.conf.j2 dest=/etc/etcd/etcd.conf
   notify:
     - restart etcd
 
-- name: Open firewalld port for etcd
-  firewalld: port=4001/tcp permanent=false state=enabled
-
-- name: Save firewalld port for etcd
-  firewalld: port=4001/tcp permanent=true state=enabled
-
-- name: Enable etcd
-  service: name=etcd enabled=yes state=started
+#- name: Open firewalld port for etcd
```

Comment:
If we create a dependency on openshift_common, we can set the openshift_firewall_allow variable in defaults/main.yml and let openshift_common handle the firewall configuration for us.  If we do that, then we should also probably rename this to openshift_etcd.  I suspect we'll probably also need to make sure that the firewall code works as we expect it to when multiple roles are assigned that depend on openshift_common.